device/isis/segmentrouting.yaml

```diff
@@ -0,0 +1,289 @@
+components:
+  schemas:
+    Isis.SegmentRouting:
+      description: |-
+        Segment Routing (SR) turns on any node to select any path (explicit or derived from IGPs SPT computations) 
+        for each of its traffic classes. The path does not depend on a hop-by-hop 
+        signaling technique (unlike LDP or RSVP). 
+        It only depends on a set of segments that are advertised by the IS-IS routing protocol. 
+        These segments act as topological sub-paths that can be combined together to form the required path.
+        Reference: https://datatracker.ietf.org/doc/html/rfc8667
+      type: object
+      properties:
+        router_capability:
+          description: |-
+            Optional IS-IS TLV named CAPABILITY, formed of multiple sub-TLVs, which allows a router to announce its
+            capabilities within an IS-IS level or the entire routing domain.        
+          $ref: '#/components/schemas/Isis.RouterCapability'
+          x-field-uid: 1
+        node_prefix:
```

Comment:
Is this really needed or could we just configure this as yet another route range with Node bit set.

Reply:
removed